.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,19 +1,42 @@
 ---
-name: Bug report
-about: Bug report
+name: Bug Report
+about: Report an unexpected behaviour that you have encountered.
 title: ''
 labels: bug
 assignees: ''
 
 ---
 
-**V version:** 
-**OS:** 
+**V Version:**
 
-**What did you do?**
+**OS:**
 
+**Source Code to reproduce the bug:**
 
-**What did you expect to see?**
+**Steps to reproduce the bug:**
+1.
+2.
 
- 
-**What did you see instead?**
+**Expected Result:**
+
+**Actual Result:**
+
+=====Delete message past this line before submitting your issue=====
+If you include source code or command line output, please put it in between code blocks like so:
+```
+fn my_v_code_should_be_surrounded_by_the_backticks() {
+
+}
+```
+
+or
+
+```
+root@v-machine$ v -o v compiler
+A C error has occurred! This shouldn't happen.
+root@v-machine$
+```
+
+You are welcome to remove the numbering on "Steps to reproduce the bug" if the bug can be reproduced by simply building your provided code.
+
+You can get your V version by typing the following in your terminal: `git describe --tags`
```

Comment:
That is not informative enough.
./v --version is better in my opinion, since it includes the commit from which v was compiled.

Reply:
On the other hand, git would be more available, while v itself may not be build in a buggy situation, so `git describe --tags` may indeed be better...

Reply:
`git describe --tags` does actually add the revision (commit hash). It outputs something that looks like `v1.2.3-150-deadbeef`.
`v1.2.3` is tag name.
`150` is the amount of commit ahead of tag.
`deadbeef` is the commit hash.

I couldn't tell you the current commit as I'm on mobile but you should be able to verify it for yourself.